src/command.rs

```diff
@@ -0,0 +1,49 @@
+use clap::{AppSettings, Parser, ValueEnum};
+use std::fmt::{Display, Formatter};
+
+#[derive(Parser)]
+#[clap(name = "cargo")]
+#[clap(bin_name = "cargo")]
+pub enum Cargo {
+    #[clap(name = "3ds")]
+    Input(Input),
+}
+
+#[derive(clap::Args)]
```

Comment:
nit: I'm a little confused why the other derive macros are imported but this one is fully qualified. Not a blocking comment, but just noticed the inconsistency.

Reply:
Missed removing that. Doing so now.